package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "literacy",
   "description": "Literate programming in JavaScript using reStructuredText.",
-  "version": "0.2.5",
+  "version": "0.3.0",
```

Comment:
Most projects prefer version bumps to be atomic, stand-alone commits.

Reply:
Fixed, thanks.